lib/server.js

```diff
@@ -139,6 +145,12 @@ function multipleBindings (mainServer, httpHandler, serverOpts, listenOptions, o
         })
 
         const secondaryServer = getServerInstance(serverOpts, httpHandler)
+        /* istanbul ignore else */
+        if (secondaryServer === mainServer) {
```

Comment:
We are just interested in looking whether the server is listening already or not to print the warning 👍 

```suggestion
        if (secondaryServer.listening) {
```